addons/web/static/tests/webclient/user_menu_tests.js

```diff
@@ -95,7 +90,7 @@ QUnit.test("can be rendered", async (assert) => {
     assert.containsOnce(target, "img.o_user_avatar");
     assert.strictEqual(
         target.querySelector("img.o_user_avatar").dataset.src,
-        "http://lordofthering/web/image?model=res.users&field=avatar_128&id=7"
+        `${getOrigin()}/web/image/res.users/7/avatar_128`
```

Comment:
prefer full assert rather than includes (I want to see the unique key in all tests)